cogs/gatekeep/gatekeep.py

```diff
@@ -193,6 +194,43 @@ async def status(self, ctx: Context):
             f"- Threshold: {threshold}\n- Days to watch: {nDays}"
         )
 
+    @_gatekeep.command(name="test", aliases=["eval", "score"])
+    @commands.guild_only()
+    @checks.mod_or_permissions(administrator=True)
+    async def testMsg(self, ctx: Context, *, msg: Optional[str] = None):
```

Comment:
Skip the `str: Optional[str] = None`, and just leave it `msg: str`, this should cause the docstring to print if you don't supply `msg`.